splice_machine/src/test/java/com/splicemachine/derby/impl/load/HdfsImportIT.java

```diff
@@ -2117,4 +2122,95 @@ public void testVarbitColumn() throws Exception {
         Assert.assertEquals(expected, result);
     }
 
+    @Test
+    public void testAnalyzeExternalTable() throws Exception {
```

Comment:
@OlegMazurov does HdfsImportIT work with the remote ITs? Do we need to ignore these added tests for the remote profile?

Reply:
I see `HdfsImportIT` is excluded in `remote_it`, so disregard this.